uber/templates/registration/form.html

```diff
@@ -278,9 +281,12 @@
 <!-- TODO: integrate this with extra stuff -->
 <div class="form-group">
     <label class="col-sm-2 control-label">Badge Price</label>
-    <div class="col-sm-6">
+    <div class="checkbox col-sm-6">
         Base Price: $<input type="text" style="width:4em" name="overridden_price" value="{{ attendee.badge_cost|round(2) }}" />
-        <input type="checkbox" name="no_override" {% if attendee.overridden_price == None %} checked {% endif %}> Automatically recalculate badge price.
+        <label>
```

Comment:
There's two ways to associate a `<label>` with an input. You can use the `for` attribute as you describe. But you can also nest the `<input>` inside the `<label>`.  This [stack overflow answer](https://stackoverflow.com/a/6293626/4504644) describing labels and inputs is pretty good.